eox_tenant/api/v1/tests/test_microsites.py

```diff
@@ -0,0 +1,130 @@
+"""
+Test microsites API v1.
+"""
+
+from django.contrib.auth.models import User
+from mock import patch
+from rest_framework import status
+from rest_framework.test import APIClient, APITestCase
+
+from eox_tenant.models import Microsite
+
+
+class MicrositeAPITest(APITestCase):
```

Comment:
could we test the format of the response ? I know we don't have a established format yet but this could help to ensure that the response will not change in the future by accident, if we are waiting a error field in a 400 response it could be nice if we verify that.
But I'm not sure 100 per cent, please let me know your thoughts

Reply:
this functionality is tested in JSONRenderer, It would be repetitive